src/views/LoadManagementConfig.vue

```diff
@@ -13,6 +13,57 @@
 					</openwb-base-alert>
 				</div>
 				<div v-else>
+					<openwb-base-button-group-input
+						title="Nicht-ladende Fahrzeuge im Lastmanagement"
```

Comment:
Der Titel ist recht lang und sorgt auch bei großen Anzeigen für einen Zeilenumbruch. Da wir eh im Bereich Lastmanagement sind, kann die Angabe meiner Meinung nach entfallen.
```suggestion
						title="Nicht-ladende Fahrzeuge"
```